CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -155,13 +155,15 @@ const char *_CFProcessPath(void) {
 }
 #endif
 
-#if DEPLOYMENT_TARGET_LINUX
+#if DEPLOYMENT_TARGET_LINUX || DEPLOYMENT_TARGET_CYGWIN
 #include <unistd.h>
+#if !DEPLOYMENT_TARGET_CYGWIN
```

Comment:
Generally we avoid the negative-sense deployment target macros. Keeping them consistently positive-sense and using #if/#else instead makes it easier to reason about them (and there are lots).